{{cookiecutter.project_name}}/.github/workflows/test.yaml

```diff
@@ -48,14 +63,28 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v5
         with:
+          python-version: ${{ matrix.env.python }}
           cache-dependency-glob: pyproject.toml
+      - name: create hatch environment
+        run: uvx hatch env create ${{ matrix.env.name }}
       - name: run tests using hatch
         env:
           MPLBACKEND: agg
           PLATFORM: ${{ matrix.os }}
           DISPLAY: :42
-        run: uvx hatch test --cover --python ${{ matrix.python }}
+        run: uvx hatch run ${{ matrix.env.name }}:run-cov
```

Comment:
Just to be sure the command names `run-cov` and `coverage` can be considered stable in hatch?